meta/Meta.cpp

```diff
@@ -7067,7 +7067,11 @@ void Meta::meta_sai_on_fdb_event_single(
      * query yet.
      */
 
-    meta_fdb_event_snoop_oid(data.fdb_entry.bv_id);
+    if (meta_fdb_event_snoop_oid(data.fdb_entry.bv_id) < 0)
+    {
+        SWSS_LOG_WARN("fdb_entry oid (snoop): ignore the mac address learning during creating switch");
```

Comment:
how do you know this is during create switch ? it can be in different scenarios

Reply:
I debugged this issue and found it occurred during create switch.
However, I can't tell whether it is reproducible in other flows.